src/templates/index-template.html

```diff
@@ -180,5 +182,6 @@
     <script src="/assets/js/animation.js"></script>
     <script src="/assets/js/scripts.js"></script>
     <script src="/assets/js/pullRefresh.js"></script>
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@babylonjs/viewer@7.28.0-alpha/dist/babylon-viewer.esm.min.js"></script>
```

Comment:
nit picking, but do we want to have it fixed on a specific version?

Reply:
Not really, but see my note in the PR description:
> NOTE 2: I'm pulling in a specific version rather than just @preview because the chunk names can change from version to version which means browser caching can break things.
I tried not using @preview while working on these changes and kept running into failed chunk downloads because of this (fixed by clearing the browser cache), so I switched to a fixed version. I imagine we could fix this with stable chunks, but that's a broader problem to solve in general.